ports/operationsteel/port.json

```
@@ -0,0 +1,29 @@
+{
+  "version": 3,
+  "name": "operationsteel.zip",
+  "items": [
+    "Operation S.T.E.E.L.sh",
+    "operationsteel/"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Operation S.T.E.E.L",
+    "porter": [
+      "MadShmupper"
+    ],
+    "desc": "A roguelike shmup with hand drawn graphics, procedurally generated levels, enemies, and pickups.",
+    "desc_md": null,
+    "inst": "Purchase at itch.io https://leanderish.itch.io/steel "OR" Steam https://store.steampowered.com/app/1489300/Operation_STEEL/ -Locate OperationSTEEL.pck and place it into gamedata folder.",
```

Syntax error, double quotes need to be escaped:
```suggestion
    "inst": "Purchase at itch.io https://leanderish.itch.io/steel \"OR\" Steam https://store.steampowered.com/app/1489300/Operation_STEEL/ -Locate OperationSTEEL.pck and place it into gamedata folder.",
```